api/buildings/management/commands/buildings.py

```diff
@@ -0,0 +1,106 @@
+from django.core.management.base import BaseCommand
+from faker import Faker
+import random
+from tqdm import trange
+from time import sleep
+from buildings.models import Building
+
+fake = Faker("ro_RO")
+
+
+class Command(BaseCommand):
+
+    help = "generate X number of Fake Buildings"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "total_buildings", type=int, help="How many buildings to create"
+        )
+        parser.add_argument(
+            "--delete",
+            action="store_true",
+            help="Delete buildings",
+        )
+        parser.add_argument(
+            "--create",
+            action="store_true",
+            help="Create new fake buildings",
+        )
+
+    def handle(self, *args, **options):
+        if options["delete"]:
+            buildings_count = Building.objects.count()
+            Building.objects.filter(
+                general_id__in=list(
+                    Building.objects.values_list("pk", flat=True)[
+                        : int(options["total_buildings"])
+                    ]
+                )
+            ).delete()
+            self.stdout.write(
+                self.style.SUCCESS(
+                    "Successfully deleted %s building(s)."
+                    % (int(buildings_count))
+                )
+            )
+        elif options["create"]:
+            for building_id in trange(int(options["total_buildings"])):
+                address = fake.address()
+                risk_category = random.choice(
+                    ["U1", "U2", "U3", "U4", "RS1", "RS2", "RS3", "RS4"]
```

Comment:
While the RS1-4 is not the right naming for the risk categories, what we have currently is not right either. Will have to implement `SeismicCategoryChoice` and change the model.

Reply:
Also, would add a `"N/A"` option (which is currently found as `"n/a"`

Reply:
Ok I will implement `SeismicCategoryChoice`. Please tell me the right categories!

Reply:
The proper categories, in the order of their importance, are the following: 
1. U1
2. U2
3. U3
4. U4
5. RS I
6. RS II
7. RS III
8. RS IV
9. N/A